src/classes/RD2_OpportunityEvaluation_TEST.cls

```diff
@@ -805,6 +805,110 @@ private class RD2_OpportunityEvaluation_TEST {
     }
 
 
+    /**
+    *Verifies that no future installment will be created for fixed length RD 
+    * when number of opened installment plus closed won installments equals number of planned installments
+    */
+    @isTest
+    private static void shouldNotCreateNextInstallmentForFixedLengthRDWhenInstallmentsReachPlannedInstallmentsCount() {
+        final Integer plannedInstallments = 5;
+        final Date startDate = Date.newInstance(2020, 1, 1);
+        Date today = Date.newInstance(2020, 3, 1);
+        RD2_ScheduleService.currentDate = today;
+
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+        RD2_OpportunityEvaluationService.skipEvaluationService = true;
+
+        npe03__Recurring_Donation__c rd = getRecurringDonationBuilder()
+            .withPlannedInstallments(plannedInstallments)
+            .withRecurringTypeFixed()
+            .withStartDate(startDate)
+            .withDayOfMonth('1')
+            .build();
+
+        insert rd;
+
+        List<Opportunity> opps = new List<Opportunity>();
+        TEST_OpportunityBuilder oppBuilder = getOpportunityBuilder(rd).withName();
+
+        opps.addAll(oppBuilder
+            .withClosedWonStage()
+            .withCloseDate(startDate)
+            .build(4)
+        );
+
+        opps.add(oppBuilder
+            .withCloseDate(today.addDays(1))
+            .withOpenStage()
+            .build()
+        );
+        
+        insert opps;
+
+        RD2_OpportunityEvaluationService.skipEvaluationService = false;
```

Comment:
The fixed-length tests in PR #5220 ([commit](https://github.com/SalesforceFoundation/NPSP/pull/5220/commits/545c8a73533ea52d526641115e8509717d4af2c7)) work without `skipEvaluationService` being adjusted. Why do we need it here?

Reply:
SkipEvaluationService is remove